cmd/kubeadm/app/apis/kubeadm/v1alpha3/zz_generated.defaults.go

```diff
@@ -28,31 +28,5 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
-	scheme.AddTypeDefaultingFunc(&ClusterConfiguration{}, func(obj interface{}) { SetObjectDefaults_ClusterConfiguration(obj.(*ClusterConfiguration)) })
```

Comment:
fixed

Reply:
> please look at the generated docs and code failures here:
> https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/68595/pull-kubernetes-verify/120724

at the beginning, what i commited looks like this:
~~~
cat zz_generated.defaults.go
// +build !ignore_autogenerated

/*
Copyright The Kubernetes Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

// Code generated by defaulter-gen. DO NOT EDIT.

package v1beta1

import (
        runtime "k8s.io/apimachinery/pkg/runtime"
)

// RegisterDefaults adds defaulters functions to the given scheme.
// Public to allow building arbitrary schemes.
// All generated defaulters are covering - they call all nested defaulters.
func RegisterDefaults(scheme *runtime.Scheme) error {
        return nil
}
~~~

and its exactly what `make generated_files` generates.
but @chuckha said this may be the cause of test failures:
![default](https://user-images.githubusercontent.com/1210201/52029640-051de480-254f-11e9-8544-88610afb7092.jpg)
so should i try to commit what exactly `make generated_files` generates again ?